index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <title>Wolf's Professional Portfolio, Mobile Version</title>
+    <title>Wolf's Professional Portfolio, Mobile Version 1</title>
     <link rel="stylesheet" href="css/style.css">
     <meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
- [ ] Your CSS style is not applied, maybe you forgot to add those classes in your CSS file, and According to the requirements, You must stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). So kindly fix that 💯

| Your Design  | Figma Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/198224506-5f8204ad-a9cf-4798-9d9f-83e1080b7de0.PNG" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/198224927-a9e1f716-ceb1-4543-8177-af86c4fb59e7.png" width="500" /> |